### DISCUSSION TITLE
osgEarth 3.5?

### DISCUSSION THREAD

**Discussion by fkane** [pelicanmapping/osgearth#2352] [Support Q&A]
Hi Glenn,

Remember back in June you pushed a bunch of fixes to the osgEarth integrations for SilverLining and Triton to get them working again?

Every week I get messages from people who are trying to use SilverLining or Triton with the "official" 3.4 release, but it doesn't work because those fixes weren't in 3.4 (and they aren't comfortable with using anything newer). It looks like you're getting a few of them here as well.

Any plans for a new release that will include those fixes? If not I'll modify our documentation with a big, boldface warning to use the latest osgEarth source and not 3.4 with our SDK's.

Thanks :)

**Comment by gwaldron** [comment]
Hi Frank - we can get that going. Are you satisfied with the SL/Triton integration in the main trunk?

**Reply by fkane** [reply]
Thank you! Yes, what's in there seems to work well... folks tend to report that things start working once they update to the latest and copy over the shader files from the osgEarthTriton directory.

The water tends to look very dark due to the default light source not having much ambient to it, but that's the only thing that seems to confuse people at this point.

**Reply by gwaldron** [reply]
Gotcha. Thanks, will keep you posted.

**Reply by gwaldron** [reply]
Frank, work is "completed" on 3.5, I will wait until next week to tag it - in the meantime feel free to test against the main branch if you have time.
